README.md

```diff
@@ -1,4 +1,5 @@
 # Operate First Website
+<img src="src/assets/opf-logo.png" width="250"/>
```

Comment:
Nit: Maybe we can use the picture (with alt text) instead of the h1? Currently it looks quite repetitive and cluttered:
![image](https://user-images.githubusercontent.com/7453394/170474508-db3e662a-7691-471b-ade1-38c3dd3b8817.png)

Reply:
Didn't see that way, but I agree, I'll add the alt